src/components/Ratings/Ratings.tsx

```diff
@@ -0,0 +1,184 @@
+    error: false,
+    mode: Mode.Input,
+    size: Size.Small,
+    layout: Layout.Aside,
+    inputOption: ['Very baasa', 'Baasa', 'OK', 'Magniv', 'Achla'], //left here for playground. have a question
+  };
+
+  getDataAttributes() {
+    const { disabled, error, size } = this.props;
+
+    return {
+      [RATINGS_DATA_KEYS.Disabled]: disabled,
+      [RATINGS_DATA_KEYS.Error]: error,
+      [RATINGS_DATA_KEYS.Size]: size,
+    };
+  }
+
+  _renderContent = () => <StarIcon />;
+
+  _renderInputOption = () => {
+    const { inputOption, value } = this.props;
+
+    return (
+      <div className={styles.labelList}>
+        {inputOption.map((el, idx) => (
+          <span
+            key={idx}
+            data-hook={RATINGS_DATA_HOOKS.InputOption}
+            className={classNames(
+              styles.inputOption,
+              `${styles.inputOption}${idx + 1}`,
+            )}
+          >
+            {el}
+          </span>
+        ))}
+        <span
+          data-hook={RATINGS_DATA_HOOKS.InputOptionCurrent}
+          className={classNames(styles.inputOption, styles.inputOptionCurrent)}
+        >
+          {inputOption[value - 1]}
+        </span>
+      </div>
+    );
+  };
+
+  _renderRatingInfo = () => {
+    const { ratingDisplay, countDisplay } = this.props;
+    const info = [];
+
+    if (ratingDisplay) {
+      info.push(ratingDisplay);
+    }
+
+    if (countDisplay) {
+      info.push(countDisplay);
+    }
+
+    return (
+      <div
+        data-hook={RATINGS_DATA_HOOKS.RatingInfo}
+        className={styles.ratingInfo}
+      >
+        {info.join(' | ')}
+      </div>
+    );
+  };
+
+  render() {
+    const {
+      value,
+      onSelect,
+      disabled,
+      error,
+      size,
+      mode,
+      inputOption,
+      layout,
+      ratingDisplay,
+      countDisplay,
+      ...rest
+    } = this.props;
+    const content = this._renderContent();
+    const ratingList = Array.from(new Array(5));
+    const ratingListLength = ratingList.length;
+    const showInputOption = inputOption && mode === Mode.Input;
+    const showRatingInfo =
+      (ratingDisplay || countDisplay) && mode === Mode.Display;
+
+    return (
+      <div
+        {...styles('root', { disabled, error, size, mode, layout }, rest)}
+        {...this.getDataAttributes()}
+        data-hook={this.props['data-hook']}
+      >
+        <div className={styles.iconList}>
+          {ratingList.map((_el, idx: number) => {
+            const humanOrder = ratingListLength - idx;
+            const checked = humanOrder <= value;
+
+            return (
+              <span
+                data-hook={RATINGS_DATA_HOOKS.IconWrapper}
+                key={idx}
+                {...styles(
+                  classNames(styles.icon, `${styles.icon}${humanOrder}`),
+                  {
+                    checked,
+                  },
+                )}
+              >
+                <CoreRadio
+                  aria-label={`${5 - idx} star`}
```

Comment:
We need to find a solution for the `aria-label`s here.
I think what we can do is:
If we have `inputOptions` then we can use the as `aria-label`'s,
If not, then lets just use numbers `1`, `2`, `3`, `4`, `5` (`aria-label={5 - idx}`)